.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Same here, we should probably add .project to .ignore file just to prevent complications.